src/views/search/search.jsx

```diff
@@ -78,7 +78,12 @@ class Search extends React.Component {
         while (term.indexOf('&') > -1) {
             term = term.substring(0, term.indexOf('&'));
         }
-        term = decodeURIComponent(term.split('+').join(' '));
+        try {
```

Comment:
Why was it safe to take out the `.split('+').join(' ')` ? Because that's part of decoding uri's anyway? Do we know why it was there? Would be good to test a few strings and confirm they decode as we expect

Reply:
I believe it is because we have already decoded the URI. Regardless, including that piece of code introduces a bug when we try searching with `+`. For example, `cat+dog` gets translated to `cat dog`.